src/components/Trade/hooks/useDonationAmountBelowMinimum.tsx

```diff
@@ -1,30 +1,41 @@
-import { thorchainAssetId } from '@shapeshiftoss/caip'
+import { ethAssetId, thorchainAssetId } from '@shapeshiftoss/caip'
 import { useMemo } from 'react'
 import { bnOrZero } from 'lib/bignumber/bignumber'
 import { SwapperName } from 'lib/swapper/api'
 import { RUNE_OUTBOUND_TRANSACTION_FEE_CRYPTO_HUMAN } from 'lib/swapper/swappers/ThorchainSwapper/constants'
-import { selectMarketDataById } from 'state/slices/marketDataSlice/selectors'
-import { useAppSelector } from 'state/store'
-import { selectDonationAmountFiat } from 'state/zustand/swapperStore/amountSelectors'
+import { selectFeeAssetByChainId } from 'state/slices/assetsSlice/selectors'
+import { selectFiatToUsdRate, selectMarketDataById } from 'state/slices/marketDataSlice/selectors'
+import { store, useAppSelector } from 'state/store'
+import { selectDonationAmountUsd } from 'state/zustand/swapperStore/amountSelectors'
+import { selectSellAsset } from 'state/zustand/swapperStore/selectors'
 import { useSwapperStore } from 'state/zustand/swapperStore/useSwapperStore'
 
 export const useDonationAmountBelowMinimum = () => {
-  const runePrice = useAppSelector(state => selectMarketDataById(state, thorchainAssetId)).price
-  const donationAmountFiat = useSwapperStore(selectDonationAmountFiat)
+  const runePriceUsd = useAppSelector(state => selectMarketDataById(state, thorchainAssetId)).price
```

Comment:
`selectMarketDataById` has prices converted to fiat (i.e not usd for international users), so the conversion will be wrong below. Instead use `selectCryptoMarketData` which has the market data `price` still in usd

Reply:
we might need to rename those selectors because this type of issue is extremely easy to create

Reply:
Stellar observation, that was turbo wrong: https://github.com/shapeshift/web/pull/4536/commits/6b9705b93adb1b714ab077743c6343cb3d82e720